src/uct/tcp/tcp_sockcm.c

```diff
@@ -152,6 +152,7 @@ static uct_iface_ops_t uct_tcp_sockcm_iface_ops = {
     .ep_check                 = (uct_ep_check_func_t)ucs_empty_function_return_unsupported,
     .ep_create                = (uct_ep_create_func_t)ucs_empty_function_return_unsupported,
     .iface_flush              = (uct_iface_flush_func_t)ucs_empty_function_return_unsupported,
+    .ep_query                 = (uct_ep_query_func_t)ucs_empty_function_return_unsupported,
```

Comment:
TCP should also support query

Reply:
@yosefe Do you mean also add ep query implementation for tcp_sockcm in this PR?

Reply:
yes, i think it should be pretty small, and it's also needed to make the gtest pass (the gtest which i asked to add)

Reply:
@yosefe Done